### PR TITLE
[Feature] submission flow improvments

### PR DESCRIPTION
**Pull request by thomasaugsten**
- Finish Texts of the submission flow
- fix JIRA bug #981
- add retry button for qr code pairing for reg token
- fix registration date in test result
- fix polling conditions 
- remove teletan storing

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Views/ExposureSubmission/ExposureSubmissionTestResultHeaderView.swift [hunk 1]
```diff
@@ -41,11 +41,20 @@ class ExposureSubmissionTestResultHeaderView: DynamicTableViewHeaderFooterView {
...
-	func configure(testResult: TestResult) {
+	func configure(testResult: TestResult, timeStamp: Int64?) {
 		setupView(testResult)
 		subTitleLabel.text = AppStrings.ExposureSubmissionResult.card_subtitle
 		titleLabel.text = localizedString(for: testResult)
 		barView.layer.backgroundColor = color(for: testResult).cgColor
+		let formatter = DateFormatter()
+		formatter.dateStyle = .medium
+		formatter.timeStyle = .none
+		formatter.timeZone = TimeZone(abbreviation: "UTC")
```
Comment (johannesrohwer) [inline src/xcode/ENA/ENA/Source/Views/ExposureSubmission/ExposureSubmissionTestResultHeaderView.swift:52]: is it safe to hardcode UTC here?

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionQRScannerViewController.swift [hunk 2]
```diff
@@ -24,6 +24,17 @@ enum QRScannerError: Error {
 	case other
 }
 
+extension QRScannerError: LocalizedError {
```
Comment (johannesrohwer) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionQRScannerViewController.swift:27]: good improvement!

**Review by johannesrohwer — APPROVED** [review]

**Review by ChristianKienle — APPROVED** [review]

**Review by melloskitten — APPROVED** [review]

**Review by ChristianKienle — CHANGES_REQUESTED** [review]
See comment

**Review by ChristianKienle — APPROVED** [review]
approved - please change the tz before merging.

**Pull request merged**